src/util/std_expr.h

```diff
@@ -2033,8 +1930,7 @@ inline void validate_expr(const object_descriptor_exprt &value)
 }
 
 
-/*! \brief TO_BE_DOCUMENTED
-*/
+/// \brief TO_BE_DOCUMENTED
```

Comment:
What is this?

Reply:
@peterschrammel I don't really now what this class represents. Do you have some more insights as to what to write here? Or @smowton?

Reply:
That should be our representation of heap-allocated objects, as far as I can tell.

Reply:
Makes sense. Added Michael's suggestion.